python/ray/experimental/client/worker.py

```diff
@@ -2,14 +2,9 @@
 import grpc
```

Comment:
I had to change the above line to `import ray.cloudpickle as cloudpickle` in order to run the client app.